### ISSUE TITLE
ImportError: cannot import name 'url_decode' from 'werkzeug.urls' when using Flask 3.0.0

### ISSUE THREAD

**Issue by ramsey-grove** [maxcountryman/flask-login#780]
After upgrading to Flask 3.0.0, the following imports no longer work in `flask_login/utils.py`:

```
from werkzeug.urls import url_decode
from werkzeug.urls import url_encode
```

**To Reproduce**
1. In a project that uses Flask 3.0.0 and Flask Login 0.6.2, include `from flask_login import LoginManager` and initialize `LoginManager`.
2. Run tests or the project
3. Observe `ImportError: cannot import name 'url_decode' from 'werkzeug.urls' `

**Expected behavior**
No errors when using LoginManager.

**Desktop (please complete the following information):**
 - OS: macOS Ventura
 - Version: 13.5.2

**Comment by davidism** [comment]
Please search for duplicates first.

**Comment by davidism** [comment]
Duplicate of #744, fixed by #746. A volunteer maintainer will work on making a release when they have time available.
